### PR TITLE
ServiceBus: Updated the resource schema for ServiceBus with latest swagger changes

### PR DESCRIPTION
**Pull request by v-Ajnava**
Updated the resource schema for ServiceBus with latest swagger changes

### CODE REVIEW

**Comment by v-Ajnava** [comment]
@Tiano2017 can you please review PR

File: schemas/2017-04-01/Microsoft.ServiceBus.json [hunk 1]
```diff
@@ -173,6 +176,55 @@
       ],
       "description": "Microsoft.ServiceBus/namespaces/disasterRecoveryConfigs"
     },
+    "namespaces_migrationConfigurations": {
```
Comment (TianoMS) [inline schemas/2017-04-01/Microsoft.ServiceBus.json]: >namespaces_migrationConfigurations [](start = 5, length = 34)

please add a test in /tests/2017-04-01

Comment (v-Ajnava) [reply schemas/2017-04-01/Microsoft.ServiceBus.json]: Resolved, added test.

File: tests/2017-04-01/Microsoft.ServiceBus.tests.json [hunk 2]
```diff
@@ -291,6 +291,34 @@
...
+        }
+      ],
+      "definition": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.ServiceBus.json#/resourceDefinitions/namespaces_migrationConfigurations",
+      "json": {
+        "type": "Microsoft.ServiceBus/invalidType",
+        "apiVersion": "2015-10-12",
+        "configName": "$default",
+        "properties": {
+        }
+      }
```
Comment (TianoMS) [inline tests/2017-04-01/Microsoft.ServiceBus.tests.json]: can we add another test case with a valid payload for  "Microsoft.ServiceBus/namespaces/migrationConfigurations"?

**Comment by Tiano2017** [comment]
@v-Ajnava One more comment about tests. Please take a look.

File: tests/2017-04-01/Microsoft.ServiceBus.tests.json (hunk 2 above)
Comment (v-Ajnava) [reply tests/2017-04-01/Microsoft.ServiceBus.tests.json]: resolved, added a valid playload test.

**Comment by v-Ajnava** [comment]
@Tiano2017 added the Valid payload test, can you please review.

**Pull request merged**